roles/idm-host-cert/README.md

```diff
@@ -0,0 +1,100 @@
+Set of Roles
+============
+
+The ansible roles found in this directory are associated with creating and configuring idm host certs.
+
+First it logs into idm, creates a session, and then registers the host. Then, it generates a private key and csr. Next, it creates the cert, retrieves the old one, and overwrites it. As a final check, it prints out the certs.
+
+Requirements
+------------
+
+An IDM accout and ansible.
+
+Role Variables
+--------------
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**idm_fqdn**| is the fully qualified domain name for the host | no | N/A |
```

Comment:
This is `yes` in the required field. Please change description to:
`FQDN of the IdM used as the CA`